cves/CVE-2016-6817.yml

```diff
@@ -141,10 +142,11 @@ interesting_commits:
     emerging themes?
     If there are no interesting commits, demonstrate that you completed this
     section by explaining what happened between the VCCs and the fix.
-  answer:
+  answer: false
```

Comment:
Wouldn't this be true if you listed a relevant commit between the VCC and fix?

Reply:
I misread the description, updated in new push